omod-common/src/main/java/org/openmrs/module/webservices/rest/web/ConversionUtil.java

```diff
@@ -58,6 +58,10 @@
 	
 	public static final String DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSSZ";
 	
+	private static final String DATE_TIME_PATTERN = "yyyy-MM-dd HH:mm";
+	
+	private static final String DATE_PATTERN = "yyyy-MM-dd";
+	
```

Comment:
Hi @mozzy11, thanks for the changes. Do we already support all the formats here https://gist.github.com/y-gagar1n/8469484 ?

Reply:
@Ruhanga  , we really do not support all of them, we only support most of them.
But would we need to support all those date formats anyway ?? cc @dkayiwa .
I would think we can merge this PR and create another ticket  for that if its necessary

Reply:
@Ruhanga , when i compare the all formats we support , with all the formats in that link , we seem to have kind of like a standard convention for the date formats. i dont think we can support something like  `dddd, dd MMMM yyyy HH:mm`, 
At a quick glance, it seems ,all the date formats we support start with `yyyy-MM-dd` ,